bedrock/firefox/templates/firefox/whatsnew/fx54/whatsnew-54.html

```diff
@@ -40,8 +46,15 @@
 {% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
 
 {% set send_to_device_message_set = 'fx-50-whatsnew' %}
-{% set app_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&amp;adgroup=iOS_App_Store&amp;creative=badge_en&amp;fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8' %}
-{% set play_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&amp;adgroup=Google_Play_Store&amp;creative=badge_en' %}
+
+{% if variation == 'c' %}
+  {% set app_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=iOS_App_Store&creative=badge_en_v2&fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8' %}
```

Comment:
I'm not sure why, but when set with these variables, the ampersands never get decoded. If you copy this link and put it directly in an `href`, however, the ampersands *do* get decoded. 🤷‍♂️